src/components/permissions/routes.ts

```diff
@@ -101,37 +92,54 @@ const routeUserPermissions = async (request: any, reply: any) => {
     })
 
   const { username } = query
-  const orgId = query.ordId ?? null
+  const orgId = query.orgId ?? null
```

Comment:
Does this need to be `  const orgId = query?.orgId ?? null`

Reply:
Yes, that's better. Thanks. 

Done b1dbdeb